<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileLoader</class>
 <widget class="QWidget" name="fileLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>147</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="2,2,1,0">
     <item>
      <widget class="QComboBox" name="comboBoxSingMult">
       <item>
        <property name="text">
         <string>Single file(s)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Directory</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Directory, recursive</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonShowFiles">
       <property name="text">
        <string>Edit list</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_28" stretch="0,2,1,1,0,0,0">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLoadFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filter to determine what files are loaded from directories.&lt;/p&gt;&lt;p&gt;Can also be used to filter subdirectories in recursive mode, eg Slide*/*.mat&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>*.mat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current file</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxFileNumber">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFileCount">
       <property name="text">
        <string>of 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFilePrev">
       <property name="text">
        <string>&amp;Prev</string>
       </property>
       <property name="shortcut">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFileNext">
       <property name="text">
        <string>&amp;Next</string>
       </property>
       <property name="shortcut">
        <string>N</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="0,10">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFilename">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,1,0,1,0">
     <item>
      <widget class="QLineEdit" name="lineEditLength">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUnits">
       <item>
        <property name="text">
         <string>Wavenumbers →</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wavenumbers ←</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wavelengths →</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wavelengths ←</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditWidth"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditHeight"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>pixels</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxSingMult</tabstop>
  <tabstop>pushButtonLoad</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonShowFiles</tabstop>
  <tabstop>lineEditLoadFilter</tabstop>
  <tabstop>spinBoxFileNumber</tabstop>
  <tabstop>pushButtonFilePrev</tabstop>
  <tabstop>pushButtonFileNext</tabstop>
  <tabstop>lineEditFilename</tabstop>
  <tabstop>lineEditLength</tabstop>
  <tabstop>comboBoxUnits</tabstop>
  <tabstop>lineEditWidth</tabstop>
  <tabstop>lineEditHeight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
